### ISSUE TITLE
Task Param values incorrect

### ISSUE THREAD

**Issue by cbouchar** [beaker-project/restraint#126]
Client seeing param values from previous task instead of values from current task.
Reproduce attempts:
* I created a task which only sets params and print them inbetween the two tasks in question and I'm seeing the same behavior.  * I tried creating my own job with the two tasks in question but it wasn't reproducible.
* Ran the questionable job and put a reserve system so I could dump the recipe XML file.  The file is correct so issue seems to be on restraint side and not beaker.
Job details are in the irc chat below.

<amahdal> hi, are <param/> supposed to appear as envvars?  because in my recipes, they do get set but it looks like most tasks have value of some params incorrect (stuck from previous task):  eg. https://beaker.engineering.redhat.com/jobs/4430266
<amahdal> if you look at morf__stage, in consecutive tasks they are all set to morf__stage=src  but the resp. param has different value
<amahdal> here's the runtest: http://pkgs.devel.redhat.com/cgit/tests/distribution/tree/jats/run/runtest.sh
<amahdal> (and i'm using local fork of wow)
<cbouchar_> amahdal: Could you be more specific which VAR is an issue and which task 
<amahdal> cbouchar_: task T:113222880 jats://redhat.com/oamg/morft//install/sut 0.3.15    has morf__stage=src  but the param says morf__stage=installsut
<amahdal> https://beaker-archive.host.prod.eng.bos.redhat.com/beaker-logs/2020/07/44302/4430266/8572153/113222880/taskout.log
<amahdal> https://beaker.engineering.redhat.com/jobs/4430266
<cbouchar_> amahdal: TY I'll take a look
<amahdal> TY! :)
<cbouchar_> amahdal:  I'm going to have to play with this.  Nothing is immediately standing out.

**Comment by cbouchar** [comment]
I wrote debug code to see if restraint is handling the variable morf__stage properly and it was.  I discussed the issue with
my peer Daniel who suggested I look to see if customer is setting environment variables via bash somehow.  If so, this will replace what restraint set.  I did find the following file which is hardcoding the target variable 'morf__stage' I was chasing.

[root@gold-4s-a ~]# grep stage /etc/profile.d/* | more
/etc/profile.d/oamg.sh:export morf__stage=src

This is is being closed since it is not a restraint issue.
